Automattic/newspack-plugin
# Clean up and update supported plugin list

### All Submissions:






### Changes proposed in this Pull Request:



This PR makes a few updates to the supported plugin list:
- Alphabetizes the list, so we know where future plugins should be inserted into the list.
- Removes `fake-plugin` from the list. If it's important that hang around for the component demo, I could just have it be available if Newspack dev mode is on.
- Adds the following commonly used/Newspack plugins to the list: Broadstreet, TablePress, AMP TablePress, Akismet, DocumentCloud, Redirection, Parse.ly, Regenerate Thumbnails, Newspack Rename Comments. If you know any I've missed that should be in the list, let me know. 

### How to test the changes in this Pull Request:

1. Go to Plugins screen. Verify the new plugins are in the Newspack list and the list works correctly.
2. Go to Components Demo. Verify no errors from the removal of fake-plugin.

### Other information:
